specification/analysisservices/resource-manager/Microsoft.AnalysisServices/2017-08-01-beta/analysisservices.json

```diff
@@ -607,6 +653,60 @@
         }
       }
     },
+    "GatewayDetails": {
+      "description": "The gateway details.",
+      "type": "object",
+      "properties": {
+        "gatewayResourceId": {
+          "type": "string",
+          "description": "Gateway resource to be associated with the server. Only supported with api versions 2017-07-14 or higher."
+        },
+        "gatewayObjectId": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Gateway object id from in the DMTS cluster for the gateway resource. Only supported with api versions 2017-07-14 or higher."
+        },
+        "dmtsClusterUri": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Uri of the DMTS cluster. Only supported with api versions 2017-07-14 or higher."
+        }
+      }
+    },
+    "GatewayListStatusLive": {
+      "description": "Status of gateway is live",
+      "type": "object",
+      "properties": {
+        "status": {
+          "type": "string",
+          "description": "Live message of list gateway."
+        }
+      }
+    },
+    "GatewayListStatusError": {
+      "description": "Status of gateway is error",
+      "type": "object",
+      "properties": {
+        "error": {
+          "$ref": "#/definitions/GatewayErrors",
+          "description": "Error of the list gateway status."
+        }
+      }
+    },
+    "GatewayErrors": {
```

Comment:
Why is this called `GatewayErrors`? It's used in a property called `error` and also by itself looks like it represents a single error.